src/components/learner-credit-management/BudgetDetailActivityTabContents.jsx

```diff
@@ -39,7 +43,10 @@ const BudgetDetailActivityTabContents = ({ enterpriseUUID, enterpriseFeatures })
   if (!isTopDownAssignmentEnabled || !subsidyAccessPolicy?.isAssignable) {
     return (
       <>
-        {!hasSpentTransactions && (<NoBnEBudgetActivity />)}
+        {!hasSpentTransactions && isEnterpriseGroupsEnabled && (
```

Comment:
When the feature flag is enabled, the `NoBnEBudgetActivity` component will still render for legacy Enterprise Offers with no spend. Groups UI should only related to new Learner Credit, not legacy Enterprise Offers).